style.css

```diff
@@ -2,14 +2,15 @@ body{
     margin: 0;
     padding: 0;
     font-family: Quicksand;
+    max-width: 100%;
 }
 nav{
```

Comment:
like the previous PR, using tagnames is generally a bad practice. Always give elements classes and use them to add style.